Mathlib/LinearAlgebra/Dimension/Basic.lean

```diff
@@ -93,6 +92,10 @@ theorem cardinal_le_rank' {s : Set M}
     (hs : LinearIndependent R (fun x => x : s → M)) : #s ≤ Module.rank R M :=
   hs.cardinal_le_rank
 
+theorem _root_.LinearIndepOn.encard_le_toENat_rank {ι : Type*} {v : ι → M} {s : Set ι}
```

Comment:
I would rather put it outside of the namespace than having to write `_root_` here.

Reply:
I think it's fine as it is. I agree it is desirable to avoid `_root_` but the alternatives (additional sectioning or less `variable` reuse) seem at least as undesirable given that this is just one lemma.